src/current/_includes/releases/v23.1/v23.1.9.md

```diff
@@ -0,0 +1,244 @@
+## v23.1.9
+
+Release Date: September 7, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-9-general-changes">General changes</h3>
+
+- Henceforth, all [cluster version upgrades](../v23.1/upgrade-cockroach-version.html) perform a [catalog](../v23.1/system-catalogs.html) health check and repair upgrade step before proceeding to other upgrade steps. [#105826][#105826]
```

Comment:
Hahahahaha

Fixed, thanks

(PS I appreciate the commit suggestions but I don't know how to use them as well as the local git commands without messing up my local repo state so that's why I'm not using them.  So I'll make these edits locally, commit, squash, and push.  Just didn't want you to think I was ignoring them)